docs/build/wasp-evm/v1.0.0-rc.6/docs/how-tos/integrate-your-exchange.mdx

```diff
@@ -0,0 +1,162 @@
+| [smr1prxvwqvwf7nru5q5xvh5thwg54zsm2y4wfnk6yk56hj3exxkg92mx20wl3s](https://explorer.shimmer.network/shimmer/addr/smr1prxvwqvwf7nru5q5xvh5thwg54zsm2y4wfnk6yk56hj3exxkg92mx20wl3s) | 0xccc7018e4fa63e5014332f45ddc8a5450da89572676d12d4d5e51c98d64155b3 |
+
+## Running a Hornet Node with Wasp
+
+### Install Hornet
+
+The easiest way to get your [Hornet](/hornet/welcome/) node up and running is using the
+provided [Docker image](/hornet/how_tos/using_docker/) which already includes the [Wasp](/wasp/running-a-node/)
+INX-plugin.
+
+### Post-Install Tasks
+
+After you have [installed your node](#install-hornet), you should make sure you:
+
+* [Configure Hornet's HTTP REST API](/hornet/how_tos/post_installation/#configuring-http-rest-api) to operate securely.
+* Review the configuration options for both
+[Hornet](/hornet/references/configuration/) and [Wasp](/wasp/configuration/).
+* [Add peers to your node](/hornet/references/peering/).
+
+### Interact with the ShimmerEVM Chain
+
+You can use [`wasp-cli`](/wasp-cli/how-tos/wasp-cli/) to safely interact with
+and [manage](wasp-cli/how-tos/chain-management/) your Wasp instance.
+
+## Creating Transactions
+
+ShimmerEVM handles transactions in the same way as standard EVM transaction with minor exceptions:
+
+- They must be signed with ShimmerEVM's ChainID (<ChainId url='https://json-rpc.evm.shimmer.network'/>).
+- The used EVM gas is converted to ISC gas before being charged to the sender.
+- EVM contracts can't receive on-ledger requests.
+
+:::warning Compatibility
+
+Please make sure that you review
+the [EVM Compatibility in IOTA Smart Contracts guide](../getting-started/compatibility.md#on-ledger-requests).
+
+:::
+
+### Determining Finality
+
+[Block times are variable in ShimmerEVM](../getting-started/compatibility.md#no-enforced-block-time). A new EVM "block"
+will be created only when an ISC block is created, and ISC does not enforce an average block time. This means that block
+times are variable; a new block will be created as soon as needed. The average block time is 2.8 seconds.
+
+## Tooling
+
+ShimmerEVM is [compatible with most wallet and tools](../getting-started/compatible-tools.md) used
+in EVM, such as [Metamask](../getting-started/compatible-tools.md#metamask), [Remix](../getting-started/compatible-tools.md#remix),
+[Hardhat](../getting-started/compatible-tools.md#hardhat) and [Ethers.js](../getting-started/compatible-tools.md#ethersjsweb3js).
+
+:::info ShimmerEVM Testnet
+
+The [ShimmerEVM Testnet](/build/networks-endpoints/#testnet-evm) is a great way to develop and test your integration
+without incurring in any risks or costs.
+
+:::
+
+## On-Chain Data
+
+ShimmerEVM support any standard way of consuming on-chain data you use for any other EVM network.
```

Comment:
This is a vague statement, that is wrong. We, for example don't support the parity API.

just state that we support the standard "go-ethereum" API